nixos/modules/services/development/lorri.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.lorri;
+  socketUnit = "lorri";
+  socketPath = "lorri/daemon.socket";
+in with lib; {
```

Comment:
I'd consider this dangerous because the scope of this is way too large. I think it's better to instead use `inherit` to bring only more frequently used attributes from `lib` into the scope.